text-generation-inference/server/text_generation_server/jetstream_pt_support/generator.py

```diff
@@ -38,6 +38,7 @@
 
 # These will do some bucketing on prefill lengths to avoid too many different sizes
 PREFILL_LENGTHS = [
+    16,
     32,
     64,
```

Comment:
Isnt it a bit too small?

Reply:
umh, it's actually aligned with [Jetstream's](https://github.com/google/JetStream/blob/93de5901a19d5271ceea7de107406b6a40f52c0c/jetstream/engine/token_utils.py#L36) buckets. But I just realized that's been extracted from the function, so I can import the constant now, I'll make the change.